src/routes/_pages/community/index.html

```diff
@@ -36,6 +36,12 @@ <h2 class="community-header">
                       pinnable="true"
                       pinIndex={3}
         />
+        <PageListItem href="/bookmarks"
+          label="Bookmarks"
+          icon="#fa-bookmark"
+          pinnable="true"
+          pinIndex={4}
+        />
```

Comment:
In the `Lists` below, we need to increment the `4` to `5`. I should really add a test for this to catch it; it's subtle because it only breaks accessibility.